Modules/Sources/DesignSystem/Foundation/Icons.xcassets/reader.follow.imageset/Contents.json

```diff
@@ -0,0 +1,16 @@
+{
+  "images" : [
+    {
+      "filename" : "reader-follow.pdf",
```

Comment:
Shouldn't the name be "reader.follow" and not "reader-follow"?

Reply:
Removed it after replacing the button with a `DSButton`.